README.rst

```diff
@@ -5,59 +5,46 @@ EasyLink
 EasyLink is a framework that allows users to build and run highly configurable
 entity resolution (ER) pipelines.
 
-Installation
-============
-
-There are a few things to install in order to use this package:
-
-- Install singularity. If this is not already installed on your system, you will likely need to request it from your system admin. Refer to https://docs.sylabs.io/guides/4.1/admin-guide/installation.html
-
-- Install graphviz via
-    
-    ``> conda install graphviz``
+.. _python_support:
```

Comment:
These `..` directives do NOT render on the github readme. They are used to reference in other docs.